[2022-09-18 03:01:58,462] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: bigquery_dag.check_query_big_query_table 2022-09-17T03:00:00+00:00 [queued]>
[2022-09-18 03:01:58,509] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: bigquery_dag.check_query_big_query_table 2022-09-17T03:00:00+00:00 [queued]>
[2022-09-18 03:01:58,511] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2022-09-18 03:01:58,513] {taskinstance.py:1018} INFO - Starting attempt 2 of 3
[2022-09-18 03:01:58,514] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2022-09-18 03:01:58,545] {taskinstance.py:1038} INFO - Executing <Task(BigQueryCheckOperator): check_query_big_query_table> on 2022-09-17T03:00:00+00:00
[2022-09-18 03:01:58,561] {standard_task_runner.py:51} INFO - Started process 28196 to run task
[2022-09-18 03:01:58,573] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'bigquery_dag', 'check_query_big_query_table', '2022-09-17T03:00:00+00:00', '--job-id', '250', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_bigquery.py', '--cfg-path', '/tmp/tmp3uhipwwf']
[2022-09-18 03:01:58,581] {standard_task_runner.py:76} INFO - Job 250: Subtask check_query_big_query_table
[2022-09-18 03:01:58,706] {logging_mixin.py:103} INFO - Running <TaskInstance: bigquery_dag.check_query_big_query_table 2022-09-17T03:00:00+00:00 [running]> on host 27377494ab8b
[2022-09-18 03:01:58,833] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Dave
AIRFLOW_CTX_DAG_ID=bigquery_dag
AIRFLOW_CTX_TASK_ID=check_query_big_query_table
AIRFLOW_CTX_EXECUTION_DATE=2022-09-17T03:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-17T03:00:00+00:00
[2022-09-18 03:01:58,840] {sql.py:87} INFO - Executing SQL check: 
           select id from githubarchive.day.20220914
        
[2022-09-18 03:01:58,890] {logging_mixin.py:103} WARNING - /home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py:120 DeprecationWarning: This method will be deprecated. Please use `BigQueryHook.get_client` method
[2022-09-18 03:01:58,900] {taskinstance.py:1396} ERROR - [Errno 2] No such file or directory: '/opt/airflow/dags/dags/support/quick-keel-213611-bfc0c6523d4e.json'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1086, in _run_raw_task
    self._prepare_and_execute_task_with_callbacks(context, task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1260, in _prepare_and_execute_task_with_callbacks
    result = self._execute_task(context, task_copy)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1300, in _execute_task
    result = task_copy.execute(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/operators/sql.py", line 88, in execute
    records = self.get_db_hook().get_first(self.sql)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/hooks/dbapi.py", line 147, in get_first
    with closing(self.get_conn()) as conn:
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 107, in get_conn
    service = self.get_service()
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 122, in get_service
    http_authorized = self._authorize()
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/common/hooks/base_google.py", line 280, in _authorize
    credentials = self._get_credentials()
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/common/hooks/base_google.py", line 251, in _get_credentials
    credentials, _ = self._get_credentials_and_project_id()
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/common/hooks/base_google.py", line 237, in _get_credentials_and_project_id
    delegates=delegates,
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/utils/credentials_provider.py", line 309, in get_credentials_and_project_id
    return _CredentialProvider(*args, **kwargs).get_credentials_and_project()
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/utils/credentials_provider.py", line 238, in get_credentials_and_project
    credentials, project_id = self._get_credentials_using_key_path()
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/utils/credentials_provider.py", line 286, in _get_credentials_using_key_path
    self.key_path, scopes=self.scopes
  File "/home/airflow/.local/lib/python3.6/site-packages/google/oauth2/service_account.py", line 234, in from_service_account_file
    filename, require=["client_email", "token_uri"]
  File "/home/airflow/.local/lib/python3.6/site-packages/google/auth/_service_account_info.py", line 72, in from_filename
    with io.open(filename, "r", encoding="utf-8") as json_file:
FileNotFoundError: [Errno 2] No such file or directory: '/opt/airflow/dags/dags/support/quick-keel-213611-bfc0c6523d4e.json'
[2022-09-18 03:01:58,929] {taskinstance.py:1440} INFO - Marking task as UP_FOR_RETRY. dag_id=bigquery_dag, task_id=check_query_big_query_table, execution_date=20220917T030000, start_date=20220918T030158, end_date=20220918T030158
[2022-09-18 03:01:59,030] {local_task_job.py:118} INFO - Task exited with return code 1
[2022-09-18 18:30:50,105] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: bigquery_dag.check_query_big_query_table 2022-09-17T03:00:00+00:00 [queued]>
[2022-09-18 18:30:50,161] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: bigquery_dag.check_query_big_query_table 2022-09-17T03:00:00+00:00 [queued]>
[2022-09-18 18:30:50,164] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2022-09-18 18:30:50,165] {taskinstance.py:1018} INFO - Starting attempt 2 of 3
[2022-09-18 18:30:50,167] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2022-09-18 18:30:50,219] {taskinstance.py:1038} INFO - Executing <Task(BigQueryCheckOperator): check_query_big_query_table> on 2022-09-17T03:00:00+00:00
[2022-09-18 18:30:50,244] {standard_task_runner.py:51} INFO - Started process 1062 to run task
[2022-09-18 18:30:50,262] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'bigquery_dag', 'check_query_big_query_table', '2022-09-17T03:00:00+00:00', '--job-id', '274', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_bigquery.py', '--cfg-path', '/tmp/tmp7og2egs1']
[2022-09-18 18:30:50,274] {standard_task_runner.py:76} INFO - Job 274: Subtask check_query_big_query_table
[2022-09-18 18:30:50,485] {logging_mixin.py:103} INFO - Running <TaskInstance: bigquery_dag.check_query_big_query_table 2022-09-17T03:00:00+00:00 [running]> on host 27377494ab8b
[2022-09-18 18:30:50,624] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Dave
AIRFLOW_CTX_DAG_ID=bigquery_dag
AIRFLOW_CTX_TASK_ID=check_query_big_query_table
AIRFLOW_CTX_EXECUTION_DATE=2022-09-17T03:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-17T03:00:00+00:00
[2022-09-18 18:30:50,633] {sql.py:87} INFO - Executing SQL check: 
           select id from githubarchive.day.20220914
        
[2022-09-18 18:30:50,684] {logging_mixin.py:103} WARNING - /home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py:120 DeprecationWarning: This method will be deprecated. Please use `BigQueryHook.get_client` method
[2022-09-18 18:30:50,692] {taskinstance.py:1396} ERROR - [Errno 2] No such file or directory: '/opt/airflow/dags/dags/support/quick-keel-213611-2877771d11bf.json'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1086, in _run_raw_task
    self._prepare_and_execute_task_with_callbacks(context, task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1260, in _prepare_and_execute_task_with_callbacks
    result = self._execute_task(context, task_copy)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1300, in _execute_task
    result = task_copy.execute(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/operators/sql.py", line 88, in execute
    records = self.get_db_hook().get_first(self.sql)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/hooks/dbapi.py", line 147, in get_first
    with closing(self.get_conn()) as conn:
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 107, in get_conn
    service = self.get_service()
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 122, in get_service
    http_authorized = self._authorize()
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/common/hooks/base_google.py", line 280, in _authorize
    credentials = self._get_credentials()
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/common/hooks/base_google.py", line 251, in _get_credentials
    credentials, _ = self._get_credentials_and_project_id()
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/common/hooks/base_google.py", line 237, in _get_credentials_and_project_id
    delegates=delegates,
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/utils/credentials_provider.py", line 309, in get_credentials_and_project_id
    return _CredentialProvider(*args, **kwargs).get_credentials_and_project()
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/utils/credentials_provider.py", line 238, in get_credentials_and_project
    credentials, project_id = self._get_credentials_using_key_path()
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/utils/credentials_provider.py", line 286, in _get_credentials_using_key_path
    self.key_path, scopes=self.scopes
  File "/home/airflow/.local/lib/python3.6/site-packages/google/oauth2/service_account.py", line 234, in from_service_account_file
    filename, require=["client_email", "token_uri"]
  File "/home/airflow/.local/lib/python3.6/site-packages/google/auth/_service_account_info.py", line 72, in from_filename
    with io.open(filename, "r", encoding="utf-8") as json_file:
FileNotFoundError: [Errno 2] No such file or directory: '/opt/airflow/dags/dags/support/quick-keel-213611-2877771d11bf.json'
[2022-09-18 18:30:50,710] {taskinstance.py:1440} INFO - Marking task as UP_FOR_RETRY. dag_id=bigquery_dag, task_id=check_query_big_query_table, execution_date=20220917T030000, start_date=20220918T183050, end_date=20220918T183050
[2022-09-18 18:30:50,781] {local_task_job.py:118} INFO - Task exited with return code 1
[2022-09-18 18:37:27,878] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: bigquery_dag.check_query_big_query_table 2022-09-17T03:00:00+00:00 [queued]>
[2022-09-18 18:37:28,081] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: bigquery_dag.check_query_big_query_table 2022-09-17T03:00:00+00:00 [queued]>
[2022-09-18 18:37:28,111] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2022-09-18 18:37:28,114] {taskinstance.py:1018} INFO - Starting attempt 2 of 3
[2022-09-18 18:37:28,120] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2022-09-18 18:37:28,356] {taskinstance.py:1038} INFO - Executing <Task(BigQueryCheckOperator): check_query_big_query_table> on 2022-09-17T03:00:00+00:00
[2022-09-18 18:37:28,416] {standard_task_runner.py:51} INFO - Started process 411 to run task
[2022-09-18 18:37:28,508] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'bigquery_dag', 'check_query_big_query_table', '2022-09-17T03:00:00+00:00', '--job-id', '299', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_bigquery.py', '--cfg-path', '/tmp/tmpu_q6kan_']
[2022-09-18 18:37:28,564] {standard_task_runner.py:76} INFO - Job 299: Subtask check_query_big_query_table
[2022-09-18 18:37:29,245] {logging_mixin.py:103} INFO - Running <TaskInstance: bigquery_dag.check_query_big_query_table 2022-09-17T03:00:00+00:00 [running]> on host 27377494ab8b
[2022-09-18 18:37:29,962] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Dave
AIRFLOW_CTX_DAG_ID=bigquery_dag
AIRFLOW_CTX_TASK_ID=check_query_big_query_table
AIRFLOW_CTX_EXECUTION_DATE=2022-09-17T03:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-17T03:00:00+00:00
[2022-09-18 18:37:29,984] {sql.py:87} INFO - Executing SQL check: 
           select id from githubarchive.day.20220914
        
[2022-09-18 18:37:30,509] {logging_mixin.py:103} WARNING - /home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py:120 DeprecationWarning: This method will be deprecated. Please use `BigQueryHook.get_client` method
[2022-09-18 18:37:36,541] {logging_mixin.py:103} WARNING - /home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py:2052 DeprecationWarning: This method is deprecated. Please use `BigQueryHook.insert_job` method.
[2022-09-18 18:37:36,637] {bigquery.py:1510} INFO - Inserting job airflow_1663526256595116_06d2bf8b43be9b79a3e89edf3b53e367
[2022-09-18 18:37:59,291] {local_task_job.py:118} INFO - Task exited with return code Negsignal.SIGKILL
[2022-09-18 19:17:40,498] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: bigquery_dag.check_query_big_query_table 2022-09-17T03:00:00+00:00 [queued]>
[2022-09-18 19:17:40,590] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: bigquery_dag.check_query_big_query_table 2022-09-17T03:00:00+00:00 [queued]>
[2022-09-18 19:17:40,591] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2022-09-18 19:17:40,596] {taskinstance.py:1018} INFO - Starting attempt 2 of 3
[2022-09-18 19:17:40,604] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2022-09-18 19:17:40,652] {taskinstance.py:1038} INFO - Executing <Task(BigQueryCheckOperator): check_query_big_query_table> on 2022-09-17T03:00:00+00:00
[2022-09-18 19:17:40,670] {standard_task_runner.py:51} INFO - Started process 2868 to run task
[2022-09-18 19:17:40,718] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'bigquery_dag', 'check_query_big_query_table', '2022-09-17T03:00:00+00:00', '--job-id', '364', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_bigquery.py', '--cfg-path', '/tmp/tmp708w8p1d']
[2022-09-18 19:17:40,730] {standard_task_runner.py:76} INFO - Job 364: Subtask check_query_big_query_table
[2022-09-18 19:17:40,894] {logging_mixin.py:103} INFO - Running <TaskInstance: bigquery_dag.check_query_big_query_table 2022-09-17T03:00:00+00:00 [running]> on host 27377494ab8b
[2022-09-18 19:17:41,088] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Dave
AIRFLOW_CTX_DAG_ID=bigquery_dag
AIRFLOW_CTX_TASK_ID=check_query_big_query_table
AIRFLOW_CTX_EXECUTION_DATE=2022-09-17T03:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-17T03:00:00+00:00
[2022-09-18 19:17:41,095] {sql.py:87} INFO - Executing SQL check: 
           select id from githubarchive.day.20220914
        
[2022-09-18 19:17:41,159] {logging_mixin.py:103} WARNING - /home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py:120 DeprecationWarning: This method will be deprecated. Please use `BigQueryHook.get_client` method
[2022-09-18 19:17:42,052] {logging_mixin.py:103} WARNING - /home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py:2052 DeprecationWarning: This method is deprecated. Please use `BigQueryHook.insert_job` method.
[2022-09-18 19:17:42,062] {bigquery.py:1510} INFO - Inserting job airflow_1663528662056468_06d2bf8b43be9b79a3e89edf3b53e367
[2022-09-18 19:17:42,553] {taskinstance.py:1396} ERROR - 403 POST https://bigquery.googleapis.com/bigquery/v2/projects/quick-keel-213611/jobs?prettyPrint=false: Access Denied: Project quick-keel-213611: User does not have bigquery.jobs.create permission in project quick-keel-213611.

(job ID: airflow_1663528662056468_06d2bf8b43be9b79a3e89edf3b53e367)

             -----Query Job SQL Follows-----             

    |    .    |    .    |    .    |    .    |    .    |
   1:
   2:           select id from githubarchive.day.20220914
   3:        
    |    .    |    .    |    .    |    .    |    .    |

(job ID: airflow_1663528662056468_06d2bf8b43be9b79a3e89edf3b53e367)

             -----Query Job SQL Follows-----             

    |    .    |    .    |    .    |    .    |    .    |
   1:
   2:           select id from githubarchive.day.20220914
   3:        
    |    .    |    .    |    .    |    .    |    .    |
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1086, in _run_raw_task
    self._prepare_and_execute_task_with_callbacks(context, task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1260, in _prepare_and_execute_task_with_callbacks
    result = self._execute_task(context, task_copy)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1300, in _execute_task
    result = task_copy.execute(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/operators/sql.py", line 88, in execute
    records = self.get_db_hook().get_first(self.sql)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/hooks/dbapi.py", line 152, in get_first
    cur.execute(sql)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 2639, in execute
    self.job_id = self.hook.run_query(sql)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 2186, in run_query
    job = self.insert_job(configuration=configuration, project_id=self.project_id)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/common/hooks/base_google.py", line 425, in inner_wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 1512, in insert_job
    job.result()
  File "/home/airflow/.local/lib/python3.6/site-packages/google/cloud/bigquery/job/query.py", line 1159, in result
    super(QueryJob, self).result(retry=retry, timeout=timeout)
  File "/home/airflow/.local/lib/python3.6/site-packages/google/cloud/bigquery/job/base.py", line 628, in result
    self._begin(retry=retry, timeout=timeout)
  File "/home/airflow/.local/lib/python3.6/site-packages/google/cloud/bigquery/job/query.py", line 1069, in _begin
    super(QueryJob, self)._begin(client=client, retry=retry, timeout=timeout)
  File "/home/airflow/.local/lib/python3.6/site-packages/google/cloud/bigquery/job/base.py", line 438, in _begin
    timeout=timeout,
  File "/home/airflow/.local/lib/python3.6/site-packages/google/cloud/bigquery/client.py", line 640, in _call_api
    return call()
  File "/home/airflow/.local/lib/python3.6/site-packages/google/api_core/retry.py", line 286, in retry_wrapped_func
    on_error=on_error,
  File "/home/airflow/.local/lib/python3.6/site-packages/google/api_core/retry.py", line 184, in retry_target
    return target()
  File "/home/airflow/.local/lib/python3.6/site-packages/google/cloud/_http.py", line 483, in api_request
    raise exceptions.from_http_response(response)
google.api_core.exceptions.Forbidden: 403 POST https://bigquery.googleapis.com/bigquery/v2/projects/quick-keel-213611/jobs?prettyPrint=false: Access Denied: Project quick-keel-213611: User does not have bigquery.jobs.create permission in project quick-keel-213611.

(job ID: airflow_1663528662056468_06d2bf8b43be9b79a3e89edf3b53e367)

             -----Query Job SQL Follows-----             

    |    .    |    .    |    .    |    .    |    .    |
   1:
   2:           select id from githubarchive.day.20220914
   3:        
    |    .    |    .    |    .    |    .    |    .    |

(job ID: airflow_1663528662056468_06d2bf8b43be9b79a3e89edf3b53e367)

             -----Query Job SQL Follows-----             

    |    .    |    .    |    .    |    .    |    .    |
   1:
   2:           select id from githubarchive.day.20220914
   3:        
    |    .    |    .    |    .    |    .    |    .    |
[2022-09-18 19:17:42,580] {taskinstance.py:1440} INFO - Marking task as UP_FOR_RETRY. dag_id=bigquery_dag, task_id=check_query_big_query_table, execution_date=20220917T030000, start_date=20220918T191740, end_date=20220918T191742
[2022-09-18 19:17:42,683] {local_task_job.py:118} INFO - Task exited with return code 1
[2022-09-18 19:34:04,385] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: bigquery_dag.check_query_big_query_table 2022-09-17T03:00:00+00:00 [queued]>
[2022-09-18 19:34:04,564] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: bigquery_dag.check_query_big_query_table 2022-09-17T03:00:00+00:00 [queued]>
[2022-09-18 19:34:04,567] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2022-09-18 19:34:04,570] {taskinstance.py:1018} INFO - Starting attempt 2 of 3
[2022-09-18 19:34:04,572] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2022-09-18 19:34:04,733] {taskinstance.py:1038} INFO - Executing <Task(BigQueryCheckOperator): check_query_big_query_table> on 2022-09-17T03:00:00+00:00
[2022-09-18 19:34:04,787] {standard_task_runner.py:51} INFO - Started process 410 to run task
[2022-09-18 19:34:04,884] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'bigquery_dag', 'check_query_big_query_table', '2022-09-17T03:00:00+00:00', '--job-id', '389', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_bigquery.py', '--cfg-path', '/tmp/tmpx8_akwpp']
[2022-09-18 19:34:04,954] {standard_task_runner.py:76} INFO - Job 389: Subtask check_query_big_query_table
[2022-09-18 19:34:06,270] {logging_mixin.py:103} INFO - Running <TaskInstance: bigquery_dag.check_query_big_query_table 2022-09-17T03:00:00+00:00 [running]> on host 27377494ab8b
[2022-09-18 19:34:08,755] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Dave
AIRFLOW_CTX_DAG_ID=bigquery_dag
AIRFLOW_CTX_TASK_ID=check_query_big_query_table
AIRFLOW_CTX_EXECUTION_DATE=2022-09-17T03:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-17T03:00:00+00:00
[2022-09-18 19:34:08,881] {sql.py:87} INFO - Executing SQL check: 
           select id from githubarchive.day.20220914
        
[2022-09-18 19:34:09,436] {logging_mixin.py:103} WARNING - /home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py:120 DeprecationWarning: This method will be deprecated. Please use `BigQueryHook.get_client` method
[2022-09-18 19:34:15,381] {logging_mixin.py:103} WARNING - /home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py:2052 DeprecationWarning: This method is deprecated. Please use `BigQueryHook.insert_job` method.
[2022-09-18 19:34:15,430] {bigquery.py:1510} INFO - Inserting job airflow_1663529655395085_06d2bf8b43be9b79a3e89edf3b53e367
[2022-09-18 19:35:15,266] {local_task_job.py:118} INFO - Task exited with return code Negsignal.SIGKILL
